src/racket/dependency-graph.rkt

```diff
@@ -56,6 +56,16 @@
 (define (get-ids dg)
   (dep-graph-id-list dg))
 
+;; dg: dep-graph
+;; Returns if there is a path from from-id to to-id
+;; Note returns #f when from-id equals to to-id
+(define (path? dg to-id from-id)
+  #;(printf "finding path from ~a to ~a~%" from-id to-id)
```

Comment:
Remove this line, unless you're planning to continue to use it in current debugging (not future debugging). Same thing applies to all the other lines I marked.